pkg/config/config.go

```diff
@@ -53,6 +54,11 @@ type Config struct {
 	WebAuthNRPIcon        string
 	WebAuthNDebug         bool
 
+	// JWT
+	Secret        string
```

Comment:
Maybe these should include the JWT prefix? I think from the caller's perspective, `config.Expiration` may be ambiguous. Same goes for Secret.

Reply:
fixed